core/src/test/scala/com/broadinstitute/dsp/Generators.scala

```diff
@@ -1,14 +1,13 @@
 package com.broadinstitute.dsp
 
 import cats.data.NonEmptyList
-import org.broadinstitute.dsde.workbench.google2.GKEModels.{KubernetesClusterId, KubernetesClusterName}
-import org.scalacheck.{Arbitrary, Gen}
-import org.broadinstitute.dsde.workbench.google2.Generators._
 import org.broadinstitute.dsde.workbench.azure.Generators.genAzureCloudContext
+import org.broadinstitute.dsde.workbench.google2.GKEModels.KubernetesClusterName
+import org.broadinstitute.dsde.workbench.google2.Generators._
+import org.scalacheck.{Arbitrary, Gen}
 
 object Generators {
-  // TODO IA-3289 When we implement Azure, make sure to add CloudService.AzureVM as an option in the line below so tests use it
-  val genCloudService: Gen[CloudService] = Gen.oneOf(CloudService.Gce, CloudService.Dataproc)
+  val genCloudService: Gen[CloudService] = Gen.oneOf(CloudService.Gce, CloudService.Dataproc, CloudService.AzureVM)
```

Comment:
I think the unit tests weren't actually testing Azure before -- now they are. :)